multiversx_sdk/core/transaction_computer.py

```diff
@@ -122,6 +126,12 @@ def _to_dictionary(self, transaction: ITransaction) -> Dict[str, Any]:
         if transaction.guardian:
             dictionary["guardian"] = transaction.guardian
 
+        if transaction.relayer:
+            dictionary["relayer"] = transaction.relayer
+
+        if len(transaction.inner_transactions):
+            dictionary["innerTransactions"] = [self._to_dictionary(transaction=tx, with_signature=with_signature) for tx in transaction.inner_transactions]
```

Comment:
Here, `with_signature` should be set to `True`. Sorry if I misunderstood.

Reply:
changed.